test_particle_in_field.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from turbopy import Simulation
+from particle_in_field import EMWave, ChargedParticle, ForwardEuler
+
+owner_data = {"N": 10, "min": 20, "max": 30, "type": "test_pusher"}
+wave_data = {"amplitude": 10, "omega": 18}
+particle_data = {"position": 26.12, "pusher": "test_pusher"}
+
+
+@pytest.fixture(name="EM_example")
+def fixture_EM():
+    Owner = Simulation({"Grid": owner_data})
```

Comment:
Instance names should not be capitalized.
```suggestion
    owner = Simulation({"Grid": owner_data})
```